api/src/test/java/org/openmrs/module/idgen/service/IdentifierSourceServiceTest.java

```diff
@@ -220,6 +220,13 @@ public void getIdentifierSourceByUuid_shouldGetSource() {
         IdentifierSource identifierSource = identifierSourceService.getIdentifierSourceByUuid("0d47284f-9e9b-4a81-a88b-8bb42bc0a903");
         Assert.assertEquals(3, identifierSource.getId().intValue());
     }
+    
+    @Test
+    public void getIdentifierSourcesByPatientIdentifier_shouldGetIdentifierSourcesByIdentifier() {

```

Comment:
The method name is a bit unclear. When you say should get identifier sources by identifier, what do you mean?

Reply:
fixed